lib/unittest.bzl

```diff
@@ -364,6 +364,25 @@ def _begin(ctx):
     """
     return struct(ctx = ctx, failures = [])
 
+def _begin_analysis_test(ctx):
```

Comment:
Even though `_begin_analysis_test` is the same as `_begin`, its doc should be different from the documentation standpoint to draw a clear distinction between the two kinds of test.

Reply:
Agreed.